templates/index.html

```diff
@@ -7,10 +7,33 @@
 
 {% block content %}
   <section class="hero">
-    <img class="illustration" src="{{ get_url(path="/img/illustrations/undraw_server.svg") | safe }}" alt="">
-
+    <!-- Old website layout. Revert when conferance is done -->
+    <!-- <img class="illustration" src="{{ get_url(path="/img/illustrations/undraw_server.svg") | safe }}" alt="">
     <p>
       {{ config.description }}
+    </p> -->
+    <!-- <img class="illustration" src="{{ get_url(path="/img/boskonf_banner.svg") | safe }}" alt=""> -->
```

Comment:
suggestion: I guess there isn't any need to comment this part out as we can revert the PR 😊

Reply:
thats true